backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2154,3 +2154,23 @@ databaseChangeLog:
                   replaceIfExists: true
                   fullDefinition: false
                   selectQuery: SELECT internal_id, created_at, updated_at, last_seen, is_deleted FROM ${database.defaultSchemaName}.api_user
+  - changeSet:
+      id: add-patient-preferences-to-person-view
+      author: nathan@skylight.digital
+      comment: Add person patient preferences to person_no_phi_view
+      changes:
+        - createView:
+            viewName: person_no_phi_view
+            remarks: A subset of the person table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT p.internal_id, p.created_at, p.created_by, p.updated_at, p.updated_by, p.is_deleted, p.organization_id, p.facility_id, p.race, p.gender, p.ethnicity, p.county, p.state, p.employed_in_healthcare, p.role, p.resident_congregate_setting, p.tribal_affiliation, get_census_dob_group(p.birth_date) AS census_dob_group, pp.preferred_language, pp.test_result_delivery FROM ${database.defaultSchemaName}.person p JOIN patient_preferences pp on pp.internal_id=p.internal_id
```

Comment:
Please make this a multi-line string—YAML supports that just fine (I don't think it even changes the checksum!) and it's much easier on the eyes.

Reply:
Done in a2135ed.